web-registry/src/pages/CreditClassDetails/CreditClassDetails.ApprovedMethodologies.tsx

```diff
@@ -15,34 +14,33 @@ const ApprovedMethodologiesList: React.FC<{
 
   const methodologies = methodologyList?.['schema:itemListElement'];
   const count = methodologies?.length;
-  const firstMethodology = methodologies?.[0];
 
   if (!count || count < 1) return null;
+
   return (
     <LineItemLabelAbove
       label="approved methodologies"
       data={
-        <Box>
-          <Body size="xl" key={firstMethodology?.['schema:name']}>
```

Comment:
Check https://deploy-preview-1369--regen-registry.netlify.app/credit-classes/C01

We had this compacted view because a credit class can have too many methodologies to list.
I suggest we keep this compacted view, but show the first 2 instead of just the first 1.

Reply:
<img width="935" alt="Screen Shot 2022-10-05 at 1 09 06 PM" src="https://user-images.githubusercontent.com/9438198/194142797-d0d15c0b-727c-4511-9e5b-b924575292af.png">

Reply:
Ok, wasn't aware we could have that many will keep the previous behavior while showing up to 2 links then.

Reply:
@mhagel Fix here: https://github.com/regen-network/regen-web/pull/1369/commits/35c96537b3d9ac4784ddd3f1b4522b5ed9a9736d